### PR TITLE
Feature/cx 3404 update camera permission screen new design

### PR DESCRIPTION
**Pull request by anuragajwani**
# Problem
Camera Permission allow pop-up was confusing users into clicking over the image.

# Solution
Implement Camera Permission screen design without the allow pop-up

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-768-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-768-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://768-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by anuragajwani** [comment]
@stefaniacardenas @kopijunkie I am waiting for Spanish translation confirmation

**Comment by anuragajwani** [comment]
> @stefaniacardenas @kopijunkie I am waiting for Spanish translation confirmation

fixed

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-768-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-768-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://768-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/CameraPermissions/Primer/style.css [hunk 1]
```diff
@@ -50,18 +51,15 @@
...
   background-color: @color-white;
-  box-shadow: 0 3*@unit-small 6*@unit-small @color-border;
   background-image: url('assets/allow.svg');
   background-repeat: no-repeat;
   background-size: 100%;
   position: relative;
   flex: 0 1 100%;
-  /* 44% is the aspect ratio of responsive div */
+  /* 90.3% is the aspect ratio of responsive div */
   /* The percentage has been calculated by dividing the original height value divided by the original width value. In this case 121/270 = 0.44 */
```
[the remark this answers is not in the document]
Comment (anuragajwani) [reply src/components/CameraPermissions/Primer/style.css]: yes, good spot will update

[the remark this answers is not in the document]
Comment (anuragajwani) [reply src/components/CameraPermissions/Primer/style.css]: fixed

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-768-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-768-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://768-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/CameraPermissions/Primer/style.css [hunk 2]
```diff
@@ -21,20 +23,19 @@
...
   font-size: @font-size-small;
-
-  border-radius: 3*@unit-small;
-  border: 1px solid @color-background;
-  margin: 0 auto 8*@unit-small;
+  margin-top: 40*@unit;
   display: flex;
   flex-direction: column;
-  max-width: 280*@unit-small;
+  max-width: 105.01*@unit-x-small;
```
Comment (stefaniacardenas) [inline src/components/CameraPermissions/Primer/style.css]: How was this value calculated?

File: src/components/CameraPermissions/Primer/index.js [hunk 3]
```diff
@@ -10,18 +10,18 @@ const Permissions = ({onNext, translate}) => (
   <div className={`${style.container} ${theme.fullHeightContainer}`}>
     <PageTitle title={translate('webcam_permissions.allow_access')} subTitle={translate('webcam_permissions.enable_webcam_for_selfie')} />
     <div className={`${theme.thickWrapper} ${style.bodyWrapper}`}>
-      <p className={style.instructions}>{translate('webcam_permissions.click_allow')}</p>
       <div className={style.image}>
-        <div className={style.graphic}>
-          <span className={style.allow}>{translate('webcam_permissions.allow')}</span>
-        </div>
+        <div className={style.graphic}></div>
```
Comment (stefaniacardenas) [inline src/components/CameraPermissions/Primer/index.js]: Do we still need this image class wrapper around `.graphic`?

File: src/components/CameraPermissions/Primer/style.css (hunk 2 above)
Comment (anuragajwani) [reply src/components/CameraPermissions/Primer/style.css]: 105.01 is the value on abstract. The unit small was not actually getting the right value on screen though but unit-x-small was

File: src/components/CameraPermissions/Primer/index.js (hunk 3 above)
Comment (anuragajwani) [reply src/components/CameraPermissions/Primer/index.js]: looks like, I am unsure though. Just made the changes that reflected in the desired design

**Pull request merged**